Docs/GroundState/Vmc.md

```diff
@@ -0,0 +1,48 @@
+# Vmc
+Variational Monte Carlo schemes to learn the ground state using stochastic reconfiguration and gradient descent optimizers.
+## Constructor
+Constructs a ``VariationalMonteCarlo`` object given a hamiltonian, 
+sampler, optimizer, and the number of samples.
+
+|          Field          |   Type   |                                                                         Description                                                                          |
+|-------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|hamiltonian              |          |The hamiltonian of the system.                                                                                                                                |
```

Comment:
@jamesETsmith great work, you wrote a lot of docs already, thanks! 

Do you have a sense why here we don't get the type for hamiltonian etc?

Reply:
@gcarleo I haven't been able to figure it out yet. I think I jumped the gun on the title of my last commit. When I create the Docs for Vmc, I get 

```bash
/projects/jasm3285/apps/pull_requests/netket/Docs/parse.py:225: UserWarning:  Found argument `sampler` in docstring that does not exist in function signature.
  ' not exist in function signature.' % arg, UserWarning)
/projects/jasm3285/apps/pull_requests/netket/Docs/parse.py:225: UserWarning:  Found argument `hamiltonian` in docstring that does not exist in function signature.
  ' not exist in function signature.' % arg, UserWarning)
/projects/jasm3285/apps/pull_requests/netket/Docs/parse.py:225: UserWarning:  Found argument `optimizer` in docstring that does not exist in function signature.
  ' not exist in function signature.' % arg, UserWarning)
```

I'm not sure why it's not finding them in the function signature, the code looks analagous to the documentation in ``pyhilbert.hpp``.

Reply:
@jamesETsmith  This problem might be due to some corner case that the parser does not handle correctly. I will try to set a side some time later today to investigate. If it is my fault, sorry for the inconvenience.

Reply:
@ooreilly not a problem! Thanks for the help.